pages/index.tsx

```diff
@@ -205,6 +206,7 @@ export default function HomePage({
   };
 
   const filterSelect = (filterType?: string) => {
+    if (viewAll === true) setViewAll(false);
```

Comment:
Can we remove this `=== true`? Honestly asking because I know js can be weird like that sometimes. But otherwise, removing it would be a bit more terse.

Reply:
Definitely, removed it in the next commit. I think I've severed my JS and Python brain to the point where I try to be as explicit as possible in the JS conditionals. Makes sense to remove since it's just a boolean